frontend/public/components/operator-hub/index.ts

```diff
@@ -0,0 +1 @@
+export const OPERATOR_HUB_CSC_NAME = 'marketplace-enabled-operators';
```

Comment:
Not necessary for this PR, but we should either move this to const.js or move the other operator hub constant here.